source/server/config/network/mongo_proxy.h

```diff
@@ -23,6 +23,10 @@ class MongoProxyFilterConfigFactory : public NamedNetworkFilterConfigFactory {
   NetworkFilterFactoryCb createFilterFactoryFromProto(const Protobuf::Message& config,
                                                       FactoryContext& context) override;
 
+  ProtobufTypes::MessagePtr createEmptyConfigProto() override {
+    return ProtobufTypes::MessagePtr{new envoy::api::v2::filter::network::MongoProxy()};
```

Comment:
Is this covered? If not can we do the strategy that I said? (Assign the return of this to a config variable than fill it manually)

Reply:
done